lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -381,6 +393,88 @@ def stop_event_loop(self):
     stop_event_loop.__doc__=FigureCanvasBase.stop_event_loop_default.__doc__
 
 
+_flow = [Gtk.Orientation.HORIZONTAL, Gtk.Orientation.VERTICAL]
+flow_types = ['horizontal', 'vertical']
+
+
+class Window(WindowBase, Gtk.Window):
```

Comment:
Personally, I don't really see what difference it makes apart from taking up extra hard drive and memory space.

At the moment we have ``backend_gtk3.Window`` why do we need ``backend_gtk3.WindowGTK3`` as well?  By renaming ``Window`` to ``WindowGTK3`` and adding ``Window = WindowGTK3`` we create both names which as I say feels like overkill to me.  Note also that under the API that we deprecate here  ``Show`` also does it like this, we don't have ``ShowGTK3``.

I don't mind renaming it, I just don't see any need for it...